.travis.yml

```diff
@@ -1,17 +1,24 @@
-language: erlang
-install:
-  - sudo apt-get update
-  - sudo apt-get install cmake doxygen python-numpy swig python-dev libatlas-base-dev libvtk5-dev libtiff4-dev libmatio-dev
-  - mkdir build; cd build; cmake
-              -DATLAS_INCLUDE_PATH:PATH=/usr/include/atlas
-              -DBUILD_SHARED:BOOL=ON
-              -DBUILD_TESTING:BOOL=ON
-              -DPYTHON_WRAP:BOOL=ON
-              -DUSE_VTK:BOOL=OFF
-              -DUSE_ATLAS:BOOL=ON
-              -DUSE_SYSTEM_MATIO:BOOL=ON
-              -DCMAKE_SKIP_RPATH:BOOL=OFF
-              ..
-  - make
+language: cpp
+
+os:
+  - linux
+  - osx
```

Comment:
Didn't do that myself yet for anything, only uploads to pypi (eg in duecredit)

On May 7, 2016 10:50:06 AM EDT, Alexandre Gramfort <notifications@github.com> wrote:
>> -  - mkdir build; cd build; cmake
>> -              -DATLAS_INCLUDE_PATH:PATH=/usr/include/atlas
>> -              -DBUILD_SHARED:BOOL=ON
>> -              -DBUILD_TESTING:BOOL=ON
>> -              -DPYTHON_WRAP:BOOL=ON
>> -              -DUSE_VTK:BOOL=OFF
>> -              -DUSE_ATLAS:BOOL=ON
>> -              -DUSE_SYSTEM_MATIO:BOOL=ON
>> -              -DCMAKE_SKIP_RPATH:BOOL=OFF
>> -              ..
>> -  - make
>> +language: cpp
>> +
>> +os:
>> +  - linux
>> +  - osx
>
>btw @yarikoptic do you have an example of a travis build that pushes
>the generated binaries to a download server? that would be neat for us
>to have nightly builds.
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/openmeeg/openmeeg/pull/71/files/3a868b262872f8ff53fbb73231a12ae12df5304a#r62417295